notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>Again, is there any reason why that one? Just going backwards from the end of the tutorial, maybe you can specifically say you are looking for Lyman Alpha emission. In that case, would it be good to include in the summary table above the min and max wavelengths for the different instrument and filter combinations? (I presume it's in hst_table, just not shown!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>